src/rules/custom-property-no-empty-line-before/README.md

```diff
@@ -0,0 +1,83 @@
+# custom-property-no-empty-lines-before
+
+Disallow empty lines before custom properties.
+
+```css
+a { 
+  --foo-bar: 1px;
+                      /* ← */
+  --foo-baz: 2px;     /* ↑ */   
+}                     /* ↑ */
+/**                      ↑
+  *              This line */
+```
+
+## Options
+
+### `true`
+
+The following patterns are considered warnings:
+
+```css
+a { 
+  
+  --foo-bar: 1px;
+  --foo-baz: 2px; 
+}
+```
+
+```css
+a {
+  --foo-bar: 1px;
+
+  --foo-baz: 2px; 
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```css
+a { 
+  --foo-bar: 1px;
+  --foo-baz: 2px; 
+}
+```
+## Optional options
+
+### `expect: ["first-after-non-custom-property-sibling"]`
+
+Require an empty line for first custom property after a non custom property.
```

Comment:
... for *the* first ...

And let's say `after anything that's not a custom property`. (Is that right? Like, after nested rules and at-rules? What about comments?)